core/src/main/kotlin/services/OrchestratorService.kt

```diff
@@ -53,17 +53,17 @@ class OrchestratorService(
         path: String?,
         jobConfig: JobConfigurations,
         jobConfigContext: String?,
-        labels: Map<String, String>?
+        labels: Map<String, String>?,
+        jwt: String?
     ): OrtRun {
         val ortRun = db.dbQuery {
             ortRunRepository.create(repositoryId, revision, path, jobConfig, jobConfigContext, labels.orEmpty())
         }
 
-        // TODO: Set the correct token.
         orchestratorSender.send(
             Message(
                 header = MessageHeader(
-                    token = "",
+                    token = jwt.orEmpty(),
```

Comment:
@oheger-bosch This change seems to be correct according to the documentation of the property, but since we never made use of it I wonder if we should remove it instead?

Reply:
I expect that sooner or later it might be necessary to check permissions in components behind `core`. But I can't come up with an example on the spot and it can probably just as easily be added back in then.

Reply:
> according to the documentation of the property

To me, it's not clearly documented whether the `token` property may be empty. Still awaiting and answer to what so I can make changes accordingly.

Reply:
The original idea behind this token was to support a transport implementation based on direct HTTP calls that use a token for authentication. As @mnonnenmacher said, we never implemented this and had never a use case for the token. Nothing within ORT Server makes use of the token, and I am pretty sure that there are places in the code were the token is not correctly propagated when another endpoint is invoked.

So, I would also conclude that the cleanest solution is to drop this property. If an actual use case comes up later, then it could be implemented in a consistent way.

Reply:
> So, I would also conclude that the cleanest solution is to drop this property. If an actual use case comes up later, then it could be implemented in a consistent way.

I'll change my PR to remove the token then (tomorrow).